can/parser.cc

```diff
@@ -238,9 +238,7 @@ void CANParser::UpdateCans(uint64_t nanos, const capnp::List<cereal::CanData>::R
     //  continue;
     //}
 
-    std::vector<uint8_t> data(dat.size(), 0);
-    memcpy(data.data(), dat.begin(), dat.size());
-    state_it->second.parse(nanos, data);
+    state_it->second.parse(nanos, dat.begin(), dat.size());
```

Comment:
Can we just pass the `capnp::Data::Reader`, or do we want to remove capnp as a dependency too?